decide_panel/src/components/Graphics.js

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from "react";
+import Api from "../services/backend";
+import { ProgressBar } from "primereact/progressbar";
+import React from "react";
+import { Pie } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
+
+
+const Graphics = () => {
+  const [errorConection, setErrorConection] = useState(null);
+  const [state, setState] = useState({
+    data: '{}',
+  });
+
+  function deleteErrorMessage() {
+    setErrorConection(null);
+  }
+
+  useEffect(() => {
+    deleteErrorMessage();
+    Api.get_votes(1)                                       //MODIFICA
+      .then((res) => setState({ data: res }))
+      .catch((error) => {
+        setErrorConection(
+          <div className="alert alert-dark">
+            <strong>Error de conexión</strong>
+            <ProgressBar
+              mode="indeterminate"
+              style={{ height: "6px" }}
+            ></ProgressBar>
+          </div>
+        );
+      });
+  }, []);
+  let json_raw = JSON.stringify(state.data);
+  let jjson = json_raw.substring(1,json_raw.length-1);
+  var jsoon= JSON.parse(jjson);
+  var tally = jsoon.tally;
+  console.log(tally);
+  
+  return (
+    <div>
+        <Pie
+          data={{
+              labels: ["Votos"],
+              datasets: [{
+  
+                  label: "",
+                  data: tally,
+                  backgroundColor: [
+                      'rgba(255, 99, 132, 0.2)',
+                      'rgba(50, 250, 250, 0.2)',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(50, 250, 250, 0.2)",
```